src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php

```diff
@@ -34,12 +36,12 @@ function let(ObjectRepository $countryRepository, FormFactoryInterface $factory)
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Bundle\AddressingBundle\Form\EventListener\BuildAddressFormSubscriber');
+        $this->shouldHaveType(BuildAddressFormSubscriber::class);
```

Comment:
@pamil exactly, so it will return `spec\Sylius\...\MyClass` what is definitely not the correct class namespace.

But in the specification stage we do not care if the class exist or not, phpspec will tell you that. Anyway the class we're about to implement shouldn't exist before we write the first spec, so using `::class` will not work, before that and if you introduce it after the class is implemented it is equal to removing this spec, because it will always pass, so doesn't change anything and is useless.